pkg/beacon/relay/groupselection/ticket_submission.go

```diff
@@ -41,7 +41,7 @@ func submitTickets(
 
 func toChainTicket(ticket *ticket) (*relaychain.Ticket, error) {
 	return &relaychain.Ticket{
-		Value: ticket.intValue(),
+		Value: new(big.Int).SetBytes(ticket.value[:]),
```

Comment:
Why don't we use `intValue` here? I think we are fine with it now as you fixed the logic there.

Reply:
Nope, the `Value` of `relaychain.Ticket` must contain the whole `value` array of `groupselection.ticket` not just the first `8` bytes returned by `intValue`

Reply:
You are right, resolving.